src/ui/StartUp.java

```diff
@@ -102,6 +139,7 @@ private void buildImages(Graphics2D g2d) {
 
             if (n == 0) {
 
+                positivePosition = i;
```

Comment:
Aici mai trebuie un pic gandit codul. Ceva nu este in regula, dar nu imi dua seama ce anume.

Reply:
Locatia imaginii pozitive e data de cifra 0 care dupa shuffle() apare aleatoriu in pozitia images[i]. Ma mai gandesc daca vrei si la alta metoda de implementare